supertokens_python/recipe/userroles/user_role_claim.py

```diff
@@ -0,0 +1,36 @@
+# Copyright (c) 2021, VRAI Labs and/or its affiliates. All rights reserved.
```

Comment:
not related to this PR, but in userroles, syncio, functions take user_context without a default value - so that needs to be fixed.